site/gatsby-site/src/utils/AlgoliaUpdater.js

```diff
@@ -204,16 +207,27 @@ class AlgoliaUpdater {
   uploadToAlgolia = async ({ language, entries }) => {
     const indexName = `instant_search-${language}`;
 
+    const featuredReplicaIndexName = indexName + '-featured';
+
     const index = this.algoliaClient.initIndex(indexName);
 
     await index.saveObjects(entries);
 
-    await index.setSettings({
-      ...algoliaSettings,
-      attributeForDistinct: 'incident_id',
-      indexLanguages: [language],
-      queryLanguages: [language],
-    });
+    index
```

Comment:
There is a missing `await` here, execution will continue and not wait for this line to finish.

To avoid this type of error it is better to not mix `async/await` syntax with `Promises` syntax.